### ISSUE TITLE
Proposal: Ansible Community to accept Matrix as an equal partner to IRC

### ISSUE THREAD

**Issue by GregSutcliffe** [ansible-community/community-topics#36]
### Summary

Whilst I have been researching, discussing & testing this for a while, it's time to formally bring this forward. In this topic, I'll summarise the key points and link to further reading.

**I suggest we have a brief chat about this tonight, but leave the main debate & vote until next week so that people have a chance to read the links below.**

The proposal
- Matrix is accepted as equal to IRC for chat
- IRC remains a valid and supported option for those who wish to use it
- Matrix is promoted as the default option for *new* community members
- We hold regular introspection with the community to ensure we're still happy with the setup

### Additional Information

Additional thoughts
- IRC is entirely acceptable but not friendly to new users
- Element (the glossy web-client from Element.io) provides a much more familiar environment for new users
- Matrix's structure allows us to stay connected with IRC (and potentially other networks)
- Further integration is possible, such as running our conferences on Matrix

More reading links:
- [My long-form thoughts on the future path for chat](https://ansible.github.io/community/posts/matrix_and_ansible.html)
- [Supporting data on network size and stability](https://emeraldreverie.org/2021/07/20/matrix-irc-size-stability/)
- [Blog posts](https://blog.ergaster.org/tags/matrix/) from Thibault Martin of the GNOME Foundation (who use Matrix, Thib has written a lot on it)
- [Blog posts](http://exple.tive.org/blarg/category/irc/) from Mick Hoye of Mozilla on their successful transition last to Matrix last year

I have already spent some time collecting feedback from various community members. I plan to hold a [video AMA] next week (Tuesday, 3pm UK time). I'll go through the way I see Matrix working for us, the feedback I've had so far, and take any more questions folks may have.

**Comment by GregSutcliffe** [comment]
Bump for more discussion / vote tonight.

The promised 3rd post on getting started with Matrix is now up on my blog: https://emeraldreverie.org/2021/08/04/getting-started-matrix/

**Comment by GregSutcliffe** [comment]
Since we should be explicit about what we're voting on, I think there are two votes to hold:

1. That we accept Matrix as an official chat platform (alongside IRC)
2. That we update our docs, etc, to promote Matrix as the default chat option for new community members

**Comment by tadeboro** [comment]
>     1. That we accept Matrix as an official chat platform (alongside IRC)

This one sounds great.

>     2. That we update our docs, etc, to promote Matrix as the default chat option for new community members

I would rather say that we offer Matrix as the first/easier/more slack-like/etc. of the two since we already decided IRC is not going anywhere. Calling something a default gives me an impression that all other options are slowly being phased out, which contradicts our previous decisions.

**Comment by GregSutcliffe** [comment]
> I would rather say that we offer Matrix as the first/easier/more slack-like/etc. of the two

I'm good with "first choice" as wording, sure. My concern is "what should a brand new user see first on the docs page?". After that, I'm happy to haggle over wording :)

**Comment by gundalow** [comment]
Voting here as I'm not around for the meeting today. 

>     1. That we accept Matrix as an official chat platform (alongside IRC)

Agreed

>     2. That we update our docs, etc, to promote Matrix as the default chat option for new community members

I agree with the above comments that Matrix should be "First Choice", rather than "default" which could imply IRC is going away.

**Comment by tadeboro** [comment]
> I'm good with "first choice" as wording, sure. My concern is "what should a brand new user see first on the docs page?". After that, I'm happy to haggle over wording :)

I agree that the first thing users should see is Matrix stuff, so you have my vote for this one. But I would avoid using the default because for me, this implies that everyone should probably be using that and not only newcomers, which is not something we want to communicate (yet).

**Comment by GregSutcliffe** [comment]
To me "default" is "what do I pick if I have no other information" while "deprecated" would imply actively moving users away from something. However, words are tricky things, and I am happy to bend here.

I have updated the wording on the vote item to match the discussion, thanks both.

**Comment by cidrblock** [comment]
+1 for ^^^

**Comment by ssbarnea** [comment]
Since two days ago I finally managed to replace my irc client (thelounge) and fully switch to matrix (app.element.io). It is still bit confusing around the bridges with libera and oftc but it works, and I no longer need to keep a container running.
